backend/app/testbot/server.py

```
@@ -0,0 +1,45 @@
+                response_text = "Invalid JSON message received."
+            # Send the answer and an end marker
+            await websocket.send(response_text)
+            await websocket.send("[END]")
+    except websockets.exceptions.ConnectionClosed:
+        print("Client disconnected")
+    except Exception as e:
+        print(f"Error in process_query: {e}")
+
+async def start_server():
+    port = 8090
+    async with websockets.serve(process_query, "0.0.0.0", port):
```

Ruff (0.8.2)

Possible binding to all interfaces

(S104)